packages/components/src/FieldGroup/_docs/FieldGroup.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { Label } from "~components/Label"
+import { FieldGroup } from "../index"
+
+const meta = {
+  title: "KAIO-Staging/FieldGroup",
+  component: FieldGroup,
+  args: {
+    children: (
+      <>
+        <Label htmlFor="id--field-1">Email</Label>
+        <input
+          className="ms-6"
+          placeholder="Native text input..."
+          type="text"
+          id="id--field-1"
+        />
+      </>
+    ),
+  },
+} satisfies Meta<typeof FieldGroup>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  parameters: {
+    docs: {
+      canvas: {
+        sourceState: "shown",
+      },
+    },
+  },
+}
+
+export const Inline: Story = {
+  args: { inline: true },
+}
```

Comment:
Probably want to do another side-by-side comparison here, since it's not entirely clear what the final affect an `inline` `FieldGroup` looks like

Reply:
Probably mimic in the stickersheet so we can tell it always works

Reply:
Not elegant looking story but figured this was the best way to show its impact on surrounding content.
![Screenshot 2023-09-28 at 11 24 53 am](https://github.com/cultureamp/kaizen-design-system/assets/36558508/63ffdd2a-7ba2-474a-9386-7df0d1ad8278)

Reply:
I figured it was to put 2 field groups side by side? Like Username and Password 😄

Reply:
Ah right, yeah that makes a lot more sense 😆

Reply:
Looks a lot less jank now 😄 
![Uploading Screenshot 2023-09-28 at 12.00.29 pm.png…]()